Arrays/Number_of_Good_Pairs.cpp

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
Thd code looks good, thid is ghd brute force solution, cloud you add comments and optimal solution as well, thanks